data/drak/drak.txt

```diff
@@ -35,23 +35,28 @@ effect "distancer flare"
 
 outfit "Drak Draining Field"
 	category "Turrets"
-	cost 1000000
+	cost 10000000
 	thumbnail "outfit/unknown"
 	"mass" 84
 	"outfit space" -84
 	"weapon capacity" -84
 	"turret mounts" -1
 	weapon
-		"turret turn" 4.
-		"velocity" 400
+		"turret turn" 180 # instant turret tracking
+		"velocity" 1 # reduced to one as to get an aura effect
 		"lifetime" 1
+		"range override" 400 # from #4522 so that the weapon still fires as intended 
```

Comment:
For the sake of removing the error, you can comment this line out until the functionality is actually added. Anyone who wants to test this needs to uncomment this line and compile #4522.
```suggestion
#		"range override" 400 # from #4522 so that the weapon still fires as intended 
```